CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing to segbo
+
+
+## Prerequisites
+
+Clone the repos
+```
+git clone --recurse-submodules https://github.com/cldf-datasets/segbo
+```
+
+Install the python dependencies:
+```
+cd segbo
+pip install -e .[test]
```

Comment:
@J22Melody I renamed the segbo python module, so you'd likely have to re-install.

Reply:
I do not quite understand what this segbo python module is for, could you explain to me?

Reply:
The main point here is to install all of the dependencies listed in `setup.py`. Apart from that, it also registers the dataset implementation as entry point:
https://github.com/cldf-datasets/segbo/blob/250a96de648652faa523c8493ddf76aea9b34a80/setup.py#L9-L13
So if you had lots of datasets installed - and in different locations of your disk, you could discover them via `cldfbench.get_datasets`:
https://github.com/cldf/cldfbench/blob/d73b57adfdbcb82b52a3a17d8d8ad3b99cbc84df/src/cldfbench/dataset.py#L259